go/chat/attachments/store.go

```diff
@@ -104,8 +105,9 @@ type S3Store struct {
 
 // NewS3Store creates a standard Store that uses a real
 // S3 connection.
-func NewS3Store(logger logger.Logger, env *libkb.Env, runtimeDir string) *S3Store {
+func NewS3Store(g *libkb.GlobalContext, logger logger.Logger, env *libkb.Env, runtimeDir string) *S3Store {
```

Comment:
We can drop passing in logger and env as long as `GlobalContext` is now coming in here

Reply:
Done.